mpf/config_spec.yaml

```diff
@@ -1445,7 +1445,13 @@ shots:
     restart_events: event_handler|event_handler:ms|None
     advance_events: event_handler|event_handler:ms|None
     hit_events: event_handler|event_handler:ms|None
+    jump_states: list|subconfig(shot_jump_states)|None
```

Comment:
This structure seems very similar to `control_events` in other device types. What are your thoughts on calling it the same, for consistency? Even though it's strictly jump-related?